docs/src/examples/components/Tree/Types/TreeInitiallyOpenExample.shorthand.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react'
+import { Tree } from '@stardust-ui/react'
+
+const items = [
+  {
+    key: '1',
+    title: 'House Lannister',
+    initialOpen: true,
```

Comment:
What is a purpose of this prop? 😕 

- we use `defaultOpen`/`open` for such things
- at the same time I see `activeItems` in props, but I don't see a controlled usage example

I suggest to remove `initialOpen` and use `activeItems` to avoid multiple patterns.